Makefile

```diff
@@ -56,7 +56,8 @@ lint:
 install-mockgen:
 	go install github.com/golang/mock/mockgen@v1.6.0
 mockgen: install-mockgen
-	mockgen -source=core/pkg/sync/http/http_sync.go -destination=core/pkg/sync/http/mock/http.go -package=syncmock
-	mockgen -source=core/pkg/eval/ievaluator.go -destination=core/pkg/eval/mock/ievaluator.go -package=evalmock
+	cd core; mockgen -source=pkg/sync/http/http_sync.go -destination=pkg/sync/http/mock/http.go -package=syncmock
+	cd core; mockgen -source=pkg/sync/grpc/grpc_sync.go -destination=pkg/sync/grpc/mock/grpc.go -package=grpcmock
```

Comment:
i had to add the `cd` into the `core` directory here due to mockgen requiring a `go.mod` file being available when generating mocks for interface aliases (i.e. the grpc client interfaces that are originally defined in another module)

Reply:
Yes, this makes sense and is a result of the latest mono-repo refactoring :)